app/src/app.config.json

```diff
@@ -1134,6 +1134,7 @@
     "downloadPromptDelay": 50,
     "downloadPromptReminderDelay": 30,
     "enableFileAutoDownload": true,
-    "fileAutoDownloadSizeThresholdInMB": 15
+    "fileAutoDownloadSizeThresholdInMB": 15,
+    "closeButtonPosition": "right"
```

Comment:
Maybe `viewerCloseButtonPosition` would be more accurate name in this case, so the user can easily know what this param does